templates/base/server/config.py

```
@@ -0,0 +1,52 @@
+from typing import List
+
+from pydantic_settings import BaseSettings, SettingsConfigDict
+from functools import lru_cache
+
+
+class Settings(BaseSettings):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)